projects/admin/src/app/routes/base-route.ts

```diff
@@ -78,4 +80,17 @@ export class BaseRoute {
     }
     return data;
   }
+
+  /**
+   * Return the redirect url with an observable
+   * @param recordMetadata object
+   * @param baseUrl string
+   * @returns observable
```

Comment:
To be more coherent with others file, it will be better to use the notation `@param [paramName] - [type] : docstring`. We also use `@return` instead of `@returns` (with some exceptions... grrr)

Reply:
done